unfoldingWord/translationCore
# Added welcome splash and re-work the structure and order of rendering containers

### This pull request addresses:

added welcome screen as main/base screen.
#1345


### How to test this pull request:

- open app welcome screen should show.
- click get started then modal should open at user tab and if you click away or close the modal the welcome screen should render behind.
- try switching tools.
- try switching projects. (which should render the welcome screen again with the modal because the tools data is cleared when switching projects)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1420)
